### PR TITLE
Jbws 4038 add log trace

### PR DESCRIPTION
**Pull request by rsearls**
Added logging to DPConduit as requested in jira

### CODE REVIEW

File: modules/addons/transports/udp/src/main/java/org/jboss/wsf/stack/cxf/addons/transports/udp/UDPConduit.java [hunk 1]
```diff
@@ -165,6 +165,7 @@ public void close() throws IOException
                   catch (Exception e)
                   {
                      //ignore
+                     LOG.warning(e.getMessage());
```
Comment (rsvoboda) [inline modules/addons/transports/udp/src/main/java/org/jboss/wsf/stack/cxf/addons/transports/udp/UDPConduit.java]: LOG.trace(e) should be the solution I think.

You are omitting stacktrace by calling just e.getMessage() and trace is sufficient level

**Comment by rsearls** [comment]
UDPConduit is not using jboss logging.  It is using java.util.logging.Logger.  In addition this class
provides a method to get the logger it has created.  I don't want to break any existing code by changing
the logger type.  And it does not seem proper to have have 2 different logger objects in this class, so I used
the existing one.   The java logger does not provide a trace function.

**Comment by jbliznak** [comment]
Then you can use java.util.logging.Logger trace equivalent / lowest level, for example:
LOG.log(Level.FINEST, e.getMessage(), e);

**Comment by rsvoboda** [comment]
I think jboss logging should be used project wide - so probably something to be fixed. @asoldano wdyt?

**Comment by rsearls** [comment]
created different branch for requested changes

**Comment by asoldano** [comment]
@rsvoboda generally speaking yes, but in this case I agree with @rsearls on not changing, as the Logger  is enforced by AbstractConduit interface of Apache CXF.

**Pull request closed without merging**